services/harmony/app/routers/router.ts

```diff
@@ -189,9 +192,10 @@ export default function router({ skipEarthdataLogin = 'false' }: RouterConfig):
   result.use(logged(cmrCollectionReader));
 
   ogcCoverageApi.addOpenApiRoutes(result);
+  ogcEdrApi.addOpenApiRoutes(result);
   result.use(collectionPrefix('wms'), service(logged(wmsFrontend)));
 
-  result.use(/^\/(wms|ogc-api-coverages)/, (req, res, next) => {
+  result.use(/^\/(wms|ogc-api-coverages|cube)/, (req, res, next) => {
```

Comment:
I don't see the utility in adding this here. I don't think anyone is going to call `/cube` without `/ogc-api-edr/<version>`, etc. And I don't think we are going to add entries for for every spatial query type.